codeception.dist.yml

```diff
@@ -16,12 +16,7 @@ params:
   - .env
 extensions:
     enabled:
-      - Tribe\Test\Codeception\Extensions\Function_Mocker
```

Comment:
Just checking - why are we removing the function_mocker?

Reply:
@stratease Not my commits. This came from updating the base with Chupacabra. I think this was @Camwyn fixing the tests for the release.

Reply:
[I believe this is the thread that explains it.](https://lw.slack.com/archives/C01H7Q57P1C/p1724076991722039?thread_ts=1724061983.361079&cid=C01H7Q57P1C)

Reply:
Thanks.